### ISSUE TITLE
Support a 'permanently airgapped' setting on hardware keystores

### ISSUE THREAD

**Issue by craigraw** [spesmilo/electrum#6080]
As originally referenced in https://github.com/spesmilo/electrum/issues/5789#issuecomment-558050237, the Coldcard (and potentially Trezor [in future](https://github.com/trezor/trezor-firmware/issues/49#issue-434502299)?) supports hardware signing by PSBT file in an airgapped scenario. 

Currently Electrum always asks for the hardware wallet to be plugged in when it is configured as a keystore. It would be good to support a "never ask again/use as airgapped" configurable setting on the keystore that recognises the airgapped use case.

**Comment by SomberNight** [comment]
Note: to add some keywords for searches, this is about https://github.com/spesmilo/electrum/blob/e5b1596b69a1c77b3d8d2b235b65e624a2383921/electrum/plugin.py#L571

> Please insert your {}
> Verify the cable is connected and that no other application is using it.
> Try to connect again?

**Comment by scarolan** [comment]
Yes please. It would be nice to be able to permanently disable this warning for folks who want to go full airgap with their device.
